bubuku/env_provider.py

```diff
@@ -63,11 +63,11 @@ def _load_instance_ips(self, lb_name: str):
 
         for instance in response['InstanceStates']:
             if instance['State'] == 'InService':
-                private_ips.append(ec2.describe_instances(
-                    InstanceIds=[instance['InstanceId']])['Reservations'][0]['Instances'][0]['PrivateIpAddress'])
+                public_ips.append(ec2.describe_instances(
+                    InstanceIds=[instance['InstanceId']])['Reservations'][0]['Instances'][0]['PublicIpAddress'])
```

Comment:
Probably it is better to somehow determine situation when public is not specified for one or several instances